### DISCUSSION TITLE
Order of Speakers

### DISCUSSION THREAD

**Discussion by k3ybladewielder** [pyannote/pyannote-audio#1184] [Q&A] [answered]
Hi there.
Recently I made some tests and the orders was SPEAKER_03, SPEAKER_00, SPEAKER_02 and SPEAKER_01. And in another test the order was SPEAKER_01, SPEAKER_00, SPEAKER_02, SPEAKER_03.

What define the order of speakers on pipelines using diarization ?

**Comment by hbredin** [comment, marked as answer]
The actual labels are meaningless and their order is random. 

Might have been {`apple`, `orange`, `banana`, `pear`} or {`Tokyo`, `Nairobi`, `Denver`, `Berlin`}.

**Reply by sneakyPad** [reply]
Hey,
Thanks for the reply. I couldn't find that information anywhere else. It's been a while since your comment. I know that it currently is still the state but is there any possibility to enforce a chronological order?
